internal/db/schema/migrations/oss/postgres/99/03_worker_authentication.up.sql

```diff
@@ -16,6 +16,23 @@ values
   ('next'),
   ('current');
 
+create table worker_auth_ca(
```

Comment:
consider following the one-row pattern from go-kms-wrapping: https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/migrations/postgres/02_version.up.sql

Reply:
along with ^ this comment, I sort of expected an insert into this table... did I miss it?

Reply:
Ahh I see what you're saying, sorry! I've made those changes

Reply:
I've added it later on line 44

Reply:
ty!